exetera/core/fields.py

```diff
@@ -143,6 +143,27 @@ def _ensure_valid(self):
         if not self._valid_reference:
             raise ValueError("This field no longer refers to a valid underlying field object")
 
+    def __getitem__(self, item:Union[list, tuple, np.ndarray]):
+        if isinstance(item, slice):
+            # TODO
```

Comment:
You can forward to the self.data __getitem__ in the general case

Reply:
The data can be acquired with `self.data[slice]` I think, this can then be put into a mem field.